pkg/prompt/selector.go

```diff
@@ -0,0 +1,429 @@
+package prompt
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"os"
+	"runtime"
+	"strings"
+	"text/template"
+
+	"github.com/chzyer/readline"
+	"github.com/juju/ansiterm"
+	"github.com/manifoldco/promptui"
+	"github.com/manifoldco/promptui/list"
+	"github.com/manifoldco/promptui/screenbuf"
+	oktetoLog "github.com/okteto/okteto/pkg/log"
+	"golang.org/x/term"
+)
+
+const (
+	esc        = "\033["
+	showCursor = esc + "?25h"
+)
+
+type OktetoSelectorInterface interface {
+	Ask() (string, error)
+}
+
+// OktetoSelector represents the selector
+type OktetoSelector struct {
+	Label string
+	items []selectorItem
+	size  int
+
+	templates *promptui.SelectTemplates
+	keys      *promptui.SelectKeys
+
+	oktetoTemplates *oktetoTemplates
+	initialPosition int
+}
+
+// selectorItem represents a selectable item on a selector
+type selectorItem struct {
+	Name   string
+	Label  string
+	Enable bool
+}
+
+func NewOktetoSelector(label string, options []string, selectedTpl string, preselected int) *OktetoSelector {
```

Comment:
Can you include a comment in the exported function please?

Reply:
i will upload a commit with this :)